src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -0,0 +1,180 @@
+package net.maizegenetics.phgv2.cli
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.validate
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.vcf.VCFFileReader
+import htsjdk.variant.vcf.VCFHeader
+import net.maizegenetics.phgv2.utils.retrieveAgcContigs
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileWriter
+
+
```

Comment:
This class would benefit from a header comment with an overview of what the code in the class provides.  In particular, it should explain what the composite and haplotype fasta options are.   The terminology section says a "composite genome" is "inferred genome based on its composite set of alternate and reference haplotypes".    Terminology sections does not mention "haplotype fasta" though from the name it sounds like the haplotype and composite fastas are both haplotype fastas.

Some detail on why/when the user should opt for one over the other would be useful.

Reply:
I added this in, but the user facing documentation will be updated by @btmonier in the future to better describe this to the people actually using the code.